multifield.module

```diff
@@ -463,7 +463,7 @@ function multifield_get_subfields() {
     else {
       $subfields = array();
       $results = array();
-      $config_names = config_get_names_with_prefix('field.instance.');
+      $config_names = config_get_names_with_prefix('field.instance.multifield.');
       foreach ($config_names as $config_name) {
         $field = config_get($config_name);
         if ($field['entity_type'] == 'multifield') {
```

Comment:
You can save yourself a few milliseconds by not doing this `if` statement. Since you are loading all config files with prefix `field.instance.multifield.` that assures that all of those instance will be attached to a `multifield` entity type by definition.

Reply:
Thanks!